docs(event-handler): remove Amplify transformers section (#6937)
awslabs/aws-lambda-powertools-python: docs/core/event_handler/appsync.md

description: Core utility
---

> Event Handler for AWS AppSync GraphQL APIs simplifies routing and processing of events in AWS Lambda functions. It allows you to define resolvers for GraphQL types and fields, making it easier to handle GraphQL requests without the need for complex VTL or JavaScript templates.

Reviewer: How about this? JavaScript/VTL templates are used for a purpose and not necessarily to make it easier or harder. They can also be used when customers using Powertools.

```suggestion 
Event Handler for AWS AppSync GraphQL APIs simplifies routing and processing of events in AWS Lambda functions by allowing you to define resolvers for specific GraphQL types and fields.
```
